src/MOIWrapper.jl

```diff
@@ -0,0 +1,373 @@
+export SCSOptimizer
+
+using MathOptInterface
+const MOI = MathOptInterface
+const CI = MOI.ConstraintIndex
+const VI = MOI.VariableIndex
+
+const MOIU = MOI.Utilities
+
+const SF = Union{MOI.SingleVariable, MOI.ScalarAffineFunction{Float64}, MOI.VectorOfVariables, MOI.VectorAffineFunction{Float64}}
+const SS = Union{MOI.EqualTo{Float64}, MOI.GreaterThan{Float64}, MOI.LessThan{Float64}, MOI.Zeros, MOI.Nonnegatives, MOI.Nonpositives, MOI.SecondOrderCone, MOI.ExponentialCone, MOI.PositiveSemidefiniteConeTriangle}
+
+struct MOISolution
+    ret_val::Int
+    primal::Vector{Float64}
+    dual::Vector{Float64}
+    slack::Vector{Float64}
+    objval::Float64
+end
+MOISolution() = MOISolution(0, # SCS_UNFINISHED
+                      Float64[], Float64[], Float64[], NaN)
+
+mutable struct Data
```

Comment:
This name could be a bit more specific :)
Add a comment describing the purpose of this struct.